src/Handler/Setup.php

```diff
@@ -27,13 +30,8 @@ class Setup
     private $dependencies;
     private $options;
 
-    public function __construct(
-        string $cwd,
-        AppName $appName,
-        EnvEnum $envName,
-        \Traversable $dependencies,
-        array $options = []
-    ) {
+    public function __construct(string $cwd, AppName $appName, EnvEnum $envName, \Traversable $dependencies, array $options = [])
```

Comment:
@chalasr : Too long is as always: 120 chars long.
If you use PHPStorm, you can configure it to do it automatically for you.

It's not much about debugging for me, only about visual pollution, and the fact we've chosen this convention at Elao (at least we did for GBA, now I apply it on every projects).